packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -492,14 +493,14 @@ class FlutterVmService {
     if (response == null) {
       return null;
     }
-    return response.json['SkSLs'] as Map<String, Object>;
+    return response.json!['SkSLs'] as Map<String, Object>;
```

Comment:
Do you need the `!` if the method returns a nullable `Map<String, Object>?`?

Reply:
changed to `response.json?['SkSLs']`